test/Common/MoistThermodynamics/runtests.jl

```diff
@@ -91,12 +91,12 @@ using LinearAlgebra
   # variables E_int, p, q_tot, compute the temperature and partitioning of the phases
   tol_T = 1e-2
   q_tot = FT(0); ρ = FT(1)
-  @test MT.saturation_adjustment(internal_energy_sat(300.0, ρ, q_tot), ρ, q_tot) ≈ 300.0
-  @test abs(MT.saturation_adjustment_NewtonsMethod(internal_energy_sat(300.0, ρ, q_tot), ρ, q_tot) - 300.0) < tol_T
+  @test MT.saturation_adjustment_SecantMethod(internal_energy_sat(300.0, ρ, q_tot), ρ, q_tot, 1e-3, 10) ≈ 300.0
+  @test abs(MT.saturation_adjustment(internal_energy_sat(300.0, ρ, q_tot), ρ, q_tot, 1e-3, 10) - 300.0) < tol_T
 
   q_tot = FT(0.21); ρ = FT(0.1)
-  @test MT.saturation_adjustment(internal_energy_sat(200.0, ρ, q_tot), ρ, q_tot) ≈ 200.0
-  @test abs(MT.saturation_adjustment_NewtonsMethod(internal_energy_sat(200.0, ρ, q_tot), ρ, q_tot) - 200.0) < tol_T
+  @test MT.saturation_adjustment_SecantMethod(internal_energy_sat(200.0, ρ, q_tot), ρ, q_tot, 1e-3, 10) ≈ 200.0
```

Comment:
`q_tot = 0.21` is an extremely (and unrealistically) high specific humidity for 200 K and 0.1 kg/m3. More realistic at these values: `q_tot = 2e-4`.